src/sentry/integrations/slack/unfurl/metric_alerts.py

```diff
@@ -63,15 +63,17 @@ def unfurl_metric_alerts(
     if not alert_rule_map:
         return {}
 
-    incident_map = {
-        i.identifier: i
-        for i in Incident.objects.filter(
-            incident_filter_query,
-            # Filter by integration organization here as well to make sure that
-            # we have permission to access these incidents.
-            organization__in=all_integration_orgs,
-        )
-    }
+    incident_map = {}
+    if bool(incident_filter_query):
```

Comment:
Just looking at the Line 37 above, where `incident_filter_query` is initialized, does that mean `bool( Q() )` will return `False`, cause if not then I think this conditional would always succeed, right?

Reply:
Okay just realized I can check this in a Python console, you're totally right `bool(Q()) == False`